libs/components/popovers/src/index.ts

```diff
@@ -24,3 +24,5 @@ export { SkyDropdownComponent as λ3 } from './lib/modules/dropdown/dropdown.com
 export { SkyPopoverContentComponent as λ5 } from './lib/modules/popover/popover-content.component';
 export { SkyPopoverComponent as λ6 } from './lib/modules/popover/popover.component';
 export { SkyPopoverDirective as λ7 } from './lib/modules/popover/popover.directive';
+export { SkyDropdownTriggerDirective as λ8 } from './lib/modules/dropdown/dropdown-trigger.directive';
+export { SkyDropdownTriggerBaseDirective as λ9 } from './lib/modules/dropdown/dropdown-trigger-base.directive';
```

Comment:
We don't need to export components that are only used internally. I think you're safe to only export the `SkyDropdownTriggerDirective` since it will be used by omnibar.